app/middleware/csrf.js

```diff
@@ -1,77 +1,44 @@
-var csrf = require('csrf')
-var session = require('../utils/session.js')
-var views = require('../utils/views.js')
-var chargeParam = require('../services/charge_param_retriever.js')
-var logger = require('winston')
-var _views = views.create()
-
-var csrfTokenGeneration = function (req, res, next) {
-  'use strict'
-  var chargeId = chargeParam.retrieve(req)
-  var chargeSession = session.retrieve(req, chargeId)
-
-  var init = function () {
```

Comment:
I see what you were talking about now these inits are silly!

Reply:
true dat